crates/types/src/vid.rs

```diff
@@ -124,12 +124,28 @@ lazy_static! {
         UnivariateKzgPCS::<E>::gen_srs_for_testing(
             &mut rng,
             // TODO what's the maximum possible SRS size?
-            checked_fft_size(200).unwrap(),
+            jf_primitives::pcs::checked_fft_size(200).unwrap(),
         )
         .unwrap()
     };
 }
 
+// By default, use SRS from Aztec's ceremony
+#[cfg(not(feature = "test-srs"))]
+lazy_static! {
+    /// SRS comment
+    static ref KZG_SRS: UnivariateUniversalParams<E> = {
+        let srs = ark_srs::kzg10::aztec20::setup(2u64.pow(20) as usize + 2)
```

Comment:
Could we perhaps remove the magic numbers here? It's unclear what they mean and where they've come from.

Reply:
That's some upperbound that we need to provide here.
Perhaps we should define a constant for it.

Reply:
sounds good, will define a constant and add code comment to explain where this number is coming from.

Reply:
done in [47870cb](https://github.com/EspressoSystems/HotShot/pull/2854/commits/47870cb78cbbc2fb81f615f134b1140201ffbc29)